app/templates/_package.json

```diff
@@ -14,7 +14,7 @@
     "node": ">=0.10.0"
   },
   "scripts": {
-    "test": "xo && <% if (nyc) { %>nyc <% } %>ava"
+    "test": "xo && <% if (nyc) { %>nyc <% if (coveralls) { %>--reporter=lcov --reporter=text <% } %><% } %>ava"
```

Comment:
I like it this way. 

I think a "coverage" script could be confusing to some new contributors that might think they need to use it to get coverage somehow. It makes the Travis script more complex, but it also keeps Travis complexities encapsulated in the Travis file.

Also, I think it's best to keep `nyc` and `nyc report` as an atomic action (by just adding `--reporter ` flags). It's possible to cause problems if you insert additional actions between those two stages. (We have seen users try to cover two individual actions with two `nyc` commands and then generate a report - and get confused because the second `nyc` command deletes the results of the first.)